app/views/layouts/_admin_dropdown.html.erb

```diff
@@ -1,178 +1,40 @@
-<li class="dropdown right">
+<%# Privilegies to check for each menu category %>
+<% anvandare = [["Användare"],[[User], [MailAlias], [Group], [Permission]]] %>
+<% post = [["Poster"], [[Election], [Council], [Document]]] %>
+<% info = [["Information"], [[Notice], [Event], [News], [BlogPost]]] %>
+<% spindel = [["Spindelman"], [[Menu], [ShortLink], [Constant], [Category]]] %>
+<% pryl = [["Pryl"], [[Rent], [Tool]]] %>
+<% ovrigt = [["Övrigt"], [[CafeShift], [Introduction], [WorkPost], [Faq], [Page], [Album, admin_gallery_albums_path], [Contact], [Meeting], [Song]]] %>
```

Comment:
Yes this would remove the requirement of having to specify the menu links as single array elements as above.